students/ian_letourneau/Lesson06/mailroom.py

```diff
@@ -0,0 +1,169 @@
+#!/usr/bin/env python3
+# Ian Letourneau
+# 5/11/2018
+# A script that stores values for donors and their donations.
+# The script can process requests from the user on various functions
+# to utilize these values
+import datetime
+
+# Build a dictionary of existing donors
+donors = {
+    "Bill Gates": [1235544.00, 456789.50, 2347899.75],
+    "Jimmy Kimmel": [456.37, 25.67, 999876.00, 2134.78, 3.14],
+    "LeBron James": [6578921.00, 3.50, 23400.00, 7234.00]
+}
+
+
+def send_thank_you(name, amount):
+    """A function that takes in no parameters, This function allows the user
+    to lookup a list of existing donors as well as add a new donor to the
+    list. Once a name is chosen, the function prompts the user for an amount
+    donated and will add the donation to that donor's donations in the
+    dictionary. Finally, the script prints a thank you letter to that donor.
+    To quit to the main menu, 'q' can be typed at any user prompt."""
+
+    # Once amount is input, add the donation to the donor key if existing
+    # or add both the donor and the donation to the dictionary if new
+    while True:
+        try:
+            amount = float(amount)
+        except ValueError:
+            raise ValueError
+            amount = input(
+                "Invalid. Please enter the amount that was donated: ")
+            continue
+        break
+
+    if name in donors.keys():
+        donors[name].append(amount)
+    else:
+        donors.setdefault(name, [])
+        donors[name].append(amount)
+
+    print('\nDear {}, we wanted to say thank you for your generous'
+          ' donation of ${:.2f}!'.format(name, amount))
+    return('Dear {}, we wanted to say thank you for your generous'
+          ' donation of ${:.2f}!'.format(name, amount))
```

Comment:
you want to avoid adding returning value in a void function just for the sake of testing.